### PR TITLE
Feature/aba/by ardit/newsfeed layout

### PR DESCRIPTION
**Pull request by ardithoxha**
@SuchenShi @jqj5405, guys can you take a quick look at the newsfeedLayout so we can merge to the completeGUI?

### CODE REVIEW

**Review by jqj5405 — COMMENTED** [review]
For the most part, newsfeed has a nice layout and functionality. I like how you store the like list information in the parent rather than the child when using onLike() in newsfeed.ts. The need for a sidebar is questionable at the moment as we have a dedicated sticky navbar in the header. It's also non-collapsible therefore bad for responsiveness in mobile-view. There are also multiple instances where div element is used in place of p element it's best to use their respective roles when possible. 

I made some sample adjustments to the color scheme in the code below for reference going forward in relation to the first admin page color theme.

**Comment by SuchenShi** [comment]
There are conflicting files, be careful when you edit them.

File: TeamProject/src/app/modules/news-feed/newsfeed-page/newsfeed-page.component.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+.storyList {
+    display: flex;
+    flex-direction: column;
+    row-gap: 50px;
+    width: 100%;
+    padding: 20px;
+    margin-left: 100px;
+    overflow-y:auto;
+}
```
[the remark this answers is not in the document]
Comment (ardithoxha) [reply TeamProject/src/app/modules/news-feed/newsfeed-page/newsfeed-page.component.css:33]: I've set the width inside component themselves with the container class selector

File: TeamProject/src/app/modules/news-feed/newsfeed-page/newsfeed-page.component.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+* {
+    box-sizing: border-box;
+}
+
+main {
+    display: flex;
+    height: calc(100vh - 75px);
+    margin-top: 75px;
+    width: 100%;
+    border: 1px solid black;
```
[the remark this answers is not in the document]
Comment (ardithoxha) [reply TeamProject/src/app/modules/news-feed/newsfeed-page/newsfeed-page.component.css:10]: I tried this and I realized that I creates awkward contrast with the white of the content images and the whine inside the comment icon

**Pull request merged**